cypress-tests/cypress/fixtures/create-payment-method-body.json

```diff
@@ -0,0 +1,35 @@
+{
+  "payment_method": "card",
+  "payment_method_type": "credit",
+  "payment_method_issuer": "Gpay",
+  "payment_method_issuer_code": "jp_hdfc",
+  "card": {
+    "card_number": "4111111145551142",
+    "card_exp_month": "03",
+    "card_exp_year": "30",
+    "card_holder_name": "Likhin"
+  },
+  "customer_id": "{{customer_id}}",
```

Comment:
This is not called from configs as this is create payment method , where I'm not create payment here , just creating a payment method

Reply:
Clarification: We should **not** hard code values in fixtures. It should only provide a structure.
Since you're defining a value for every field, this is bound to cause issues in the coming days.

Creating a payment method should be specific to use cases unless we're sure that this fixture will only be used for cards or when using for other payment methods, every value should be updated.